src/main/java/oss/fosslight/service/impl/PartnerServiceImpl.java

```diff
@@ -10,6 +10,7 @@
 import java.util.List;
 import java.util.Map;
 
+import com.google.gson.Gson;

```

Comment:
[Question]
I don't think Gson is used in this class, so what's the reason for importing it?

Reply:
I deleted unused packages